src/components/IconButton/styles/IconButton.module.css

```diff
@@ -9,8 +9,16 @@
     cursor: pointer;
     padding: 0;
     vertical-align: middle;
+    border-radius: 4px;
```

Comment:
I suggest `border-radius: 2px;` to try to match the outer `border-radius` closer.